packages/osd-opensearch-archiver/src/lib/archives/parse.test.ts

```diff
@@ -120,7 +120,8 @@ describe('opensearchArchiver createParseArchiveStreams', () => {
           ] as [Readable, ...Writable[]]);
           throw new Error('should have failed');
         } catch (err) {
-          expect(err.message).to.contain('Unexpected number');
+          const { message } = err as Error;
```

Comment:
I know why you did this, I just hate that there isnt a nicer way of handling this :(

Reply:
Yeah, agreed. For actual error handling, I'd definitely go with a type guard, but for tests, the assertion seems a little cleaner (and preferable to a `@ts-ignore`, in my opinion), but I could easily be swayed to some other preference.